### DISCUSSION TITLE
Arduino Uno  - change input pin

### DISCUSSION THREAD

**Discussion by yabba222** [ukw100/IRMP#87] [Q&A]
Hi!
How change input pin from 3 to 8 - my ir receiver is connect to PB0 ( 8 ) arduino uno pin.
I don't see in #define line my board (UNO) in PinDefinitionsAndMore.h.
Any help will be nice :)

**Comment by ArminJo** [comment]
RTFM 😀

**Comment by yabba222** [comment]
Did you close it because you realized that I didn't read the manual? Epic.
If it was clearly described in the manual, I wouldn't be asking here?
The forum is supposed to be about solving problems, not dismissing people
asking questions.

**Comment by yabba222** [comment]
From manual:
"Pin usage
You may use every pin for input or output, just define it like #define IRMP_INPUT_PIN 2 and #define IRSND_OUTPUT_PIN 3. The PWM of the output pin is generated by software bit banging."

It is not true - I change all line  #define IRMP_INPUT_PIN xx to  #define IRMP_INPUT_PIN 8 and after compile IR input not working.

Any sugesstion or next briliant answer ?

**Reply by ArminJo** [reply]
So it must obviously be the library which has a bug!
